Controller/Shipping/Prefetch.php

```diff
@@ -65,8 +66,14 @@ class Prefetch extends Action
      */
     private $configHelper;
 
-    // GeoLocation API endpoint
-    private $locationURL = "http://freegeoip.net/json/%s";
+    /**
+     * @var CacheInterface
+     */
+    private $cache;
+
+    // GeoLocation API
+    const   API_KEY = '18beed19bdef35bf26f49f2e53c63842';
```

Comment:
I'm little uncomfortable putting this in public github repo, since we're charged based on number of request limit. can you add config in admin for this?

Reply:
Adding